paimon-core/src/main/java/org/apache/paimon/append/AppendOnlyCompactionTask.java

```diff
@@ -69,7 +69,7 @@ public CommitMessage doCompact(AppendOnlyCompactManager.CompactRewriter rewriter
                 new CompactIncrement(compactBefore, compactAfter, Collections.emptyList());
         return new CommitMessageImpl(
                 partition,
-                0, // bucket 0 is temp bucket for non-bucket table
+                0, // bucket 0 is temp bucket for unaware-bucket table
```

Comment:
To be compatible with previous design, we put these record in 'bucket = 0'. It's the temp solution for this. Maybe unaware of bucket only need one directory 'data'. Because there is no bucket concept.